frontend/src/scenes/me/Settings/index.tsx

```diff
@@ -20,15 +20,17 @@ function _MySettings(): JSX.Element {
         <div style={{ marginBottom: 128 }}>
             <PageHeader title="My Settings" />
             <Card>
-                <h2 id="password" className="subtitle">
-                    Change Password
-                </h2>
-                <ChangePassword />
-                <Divider />
-                <h2 id="personal-api-keys" className="subtitle">
-                    Personal API Keys
-                </h2>
-                <PersonalAPIKeys />
+                <span className="ph-no-capture">
```

Comment:
This one I'm less opposed to, but I do think we could sanitize the API keys specifically & the password input instead of the whole thing.